### ISSUE TITLE
A regular plugin version

### ISSUE THREAD

**Issue by vivi-gomez** [Acly/krita-ai-tools#18]
Is there any plan to become a regular plugin installation?
This is a great project and very useful. However, its installation it is complicated for a regular Krita app image with other plugin.

**Comment by Acly** [comment]
What is a "regular" plugin? If you mean Python plugin, that is limited and does not allow adding new tools (for example).

**Comment by MurzNN** [comment]
Yeah, a simplified installation would be great. If no tools can be added, maybe make just a separate docker with the plugin functionality? Here is an example of an AI plugin that works as a regular plugin: https://github.com/Acly/krita-ai-diffusion/

**Comment by Acly** [comment]
It's not a good option, apart from bad usability, the tools also inherit a lot of functionality (including all the tool options) from the other selection tools in krita. A python plugin would have to reimplement all that by hand.

Technically its possible to use the python plugin to load the SO from anywhere, but I tried this before and it didn't work (probably python plugin is loaded too late).

**Comment by txirrindulari** [comment]
> What is a "regular" plugin? If you mean Python plugin, that is limited and does not allow adding new tools (for example).

Well, I am just a user, in love with Krita and I amazed by krita-ai-tools. 
However, I am not able to make the installation coexist by opening the app file, installing and packaging it again to also have krita-ai-diffusion installed, as @MurzNN mentions.
I don't know how it should be reprogrammed, but it's actually difficult to understand the installation.

Moreover, I do not why but I can use it only with the CPU, the GPU give me an error (can't say right now exactly what is problem)

**Comment by txirrindulari** [comment]
> I don't see how this is related to using krita-ai-diffusion? It lives in a completely different place (usually `~/.local/share/krita`) and will work with the original AppImage or the modified version.
> 
> I know installation on Linux isn't ideal, but realistically the way to improve it is to get the feature into Krita.
> 
> > Moreover, I do not why but I can use it only with the CPU, the GPU give me an error (can't say right now exactly what is problem)
> 
> Using GPU on Linux requires installing a bunch of NVIDIA stuff, none of which is available as packages. It's not worth the effort.

That is the thing. I could not manage to have both things together. I am running two instances of Krita in order to use both things.
My GPU is working fine with all other software, I thought I did a wrong installation of krita-ai-tools since I only able to use the cpu.
Thank you for your support

**Comment by moonwhaler** [comment]
I, too, would love this, but I understand the sentiment and the effort involved. 

Currently, I am (Linux) distro hopping a bit, which came to a stop with Arch. Not using Flatpaks solved many issues I had with using this project. In fact, it was almost as easy as in Windows.

What Iwould really be a shame, if this project would be abandoned. It's a real help!

**Comment by yen-shi-lun** [comment]
Thank you very much for developing this plugin, 5.2.3 works fine

**Comment by xiaojianan123** [comment]
> 非常感谢您开发这个插件，5.2.3 运行良好

Why Can't I run
